docker/develop/Dockerfile

```diff
@@ -0,0 +1,188 @@
+FROM ubuntu:18.04
+LABEL org.opencontainers.image.authors="daniel.p.carbone@vumc.org"
+LABEL org.opencontainers.image.url="https://github.com/all-of-us/curation"
+LABEL org.opencontainers.image.documentation="https://github.com/all-of-us/curation"
+LABEL org.opencontainers.image.licenses="MIT"
+LABEL org.opencontainers.image.title="All of Us Curation base development image"
+LABEL org.opencontainers.image.description="Base development container image used by the All of Us Curationm team"
+
+### DESCRIPTION
+
+# this image's intent is to provide a universal base image upon which all subsequent development
+# images can be derived
+
+### CONTAINER BUILD
+
+## OS INIT
+# run the base container init first, as it is highly unlikely to change.
+# this allows us to modify subsequent steps without initating a full image rebuild
+
+# install deps and link "python" and "python3" execs to 3.7
+RUN apt update \
+    && apt upgrade -y \
+    && apt install -y \
+      git \
+      python3.7-dev \
+      python3.7-venv \
+      python3-pip \
+      python3-wheel \
+      wget \
+    && update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 1 \
+    && update-alternatives --install /usr/bin/python python /usr/bin/python3.7 1
+
+## BUILD ARGS
+
+# expected to be defined via either docker-compose or as docker build --build-arg
+ARG UID
+ARG GID
+
+# this will be the name of our in-container user
+ARG CURATION_USER="curation"
+ARG CURATION_HOME="/home/${CURATION_USER}"
+
+# gsdk args
+ARG GSDK_VERSION
```

Comment:
Was there a particular reason defualts were excluded (Understanding that `docker-compose.yaml`'s values take precedence over what would be defaulted)?

Reply:
Hmm, are you asking why I didn't define a default for some of the ARG's?

It boils down to this:

1. This container was expected to be run using compose
2. Having it defined statically inside the dockerfile AND inside the compose yaml means that updating this to a newer version now requires updating two files rather than one.

IMO if it were moved into the Dockerfile, it should be removed from the compose yaml.